tests/core/test_vad_gating.py

```
@@ -345,13 +348,159 @@ def test_create_peer_vad_gating_uses_helper_defaults():
+    gating = create_peer_vad_gating(
+        SequenceVadEngine(probs=probs),
+        sample_rate_hz=16000,
+        ring_buffer_ms=500,
+        hangover_ms=500,
+    )
+
+    events_by_chunk = []
+    for index in range(len(probs)):
+        events_by_chunk.append(
+            gating.process_chunk(chunk_samples(float(index + 1), n=gating.chunk_samples))
+        )
```

Ruff (0.16.1)

[warning] Use a list comprehension to create a transformed list

(PERF401)